src/commands/ecr-login.yml

```diff
@@ -1,6 +1,14 @@
 description: "Authenticate into the Amazon ECR service"
 
 parameters:
+  account-url:
```

Comment:
I think the `build-and-push-image` command (see: https://github.com/CircleCI-Public/aws-ecr-orb/blob/d26abc65b176b6a994d2045fc90be276a96b89e2/src/commands/build-and-push-image.yml#L144) should also be updated so that `account-url` is passed to the `ecr-login` command

For example:

```
 - ecr-login:
      profile-name: <<parameters.profile-name>>
      region: <<parameters.region>>
      account-url: <<parameters.account-url>>
```